LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2018 Userspace Containerization team at Red Hat
```

Comment:
Do we want to use `Userspace Containerization`?

Is the year correct?

Reply:
Uh, that was a copy from `packit-service`. Fixed.